Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Part 1: Common set up. This can be eventually separated into a shared base Open Targets image.
+
+FROM python:3.10
+
+# Suppress interactive prompts.
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install utilities required by Spark scripts.
+RUN apt -qq update && apt -qq install -y procps tini libjemalloc2
+
+# Enable jemalloc2 as default memory allocator.
+ENV LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so.2
+
+# Create the 'spark' group/user.
+RUN groupadd -g 1099 spark
+RUN useradd -u 1099 -g 1099 -d /home/spark -m spark
+
+# Prepare for additional Python package installation.
+ENV PIP_PACKAGES=/home/spark/packages
+RUN mkdir -p "${PIP_PACKAGES}"
+
+
+
+# Part 2: ot-release-metrics specific setup.
+
+# Install main requirements.
+COPY requirements.txt /home/spark/requirements.txt
+RUN python3 -m pip install -q --target "${PIP_PACKAGES}" -r /home/spark/requirements.txt
+
+# Install PIS and its requirements.
+ENV PIS_PATH=/home/spark/platform-input-support
+RUN python3 -m pip install -q --target "${PIP_PACKAGES}" configargparse strenum jsonpickle google-cloud-storage addict yapsy
+RUN git clone -b master https://github.com/opentargets/platform-input-support $PIS_PATH
```

Comment:
I'd pin the PIS version to be used

Reply:
In theory yes, but unfortunately PIS doesn't appear to be consistently versioned :-/ We can't pin a [release](https://github.com/opentargets/platform-input-support/releases) or a [tag](https://github.com/opentargets/platform-input-support/tags), and pinning down a specific commit would be very uninformative and even dangerous. So for now I'm afraid we don't have much choice!